cassandra-persistence/src/main/java/com/netflix/conductor/cassandra/config/CassandraProperties.java

```diff
@@ -48,7 +48,7 @@ public class CassandraProperties {
     private int shardSize = 100;
 
     /** The replication strategy with which to configure the keyspace */
-    private String replicationStrategy = "SimpleStrategy";
+    private String replicationStrategy = "NetworkTopologyStrategy";
```

Comment:
We can keep the default to simple for local setup. and we can pass this from config for staging\prod?

Reply:
Can I keep this NetworkTopologyStrategy as default strategy and and pass SimpleStrategy in dev config files?

Reply:
It may fail local setup. lets use this from config.